src/type/directives.js

```diff
@@ -52,14 +66,16 @@ export const GraphQLIncludeDirective = new GraphQLDirective({
   description:
     'Directs the executor to include this field or fragment only when ' +
     'the `if` argument is true.',
+  locations: [
```

Comment:
The list of locations should also include `DirectiveLocation.FRAGMENT_DEFINITION` according to the tests and previous behavior, isn't it?